Formula/avrdude.rb

```diff
@@ -30,46 +31,37 @@ class Avrdude < Formula
   end
 
   head do
-    url "https://svn.savannah.nongnu.org/svn/avrdude/trunk/avrdude"
+    url "https://github.com/avrdudes/avrdude.git", branch: "main"
 
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
+    depends_on "cmake" => :build
   end
 
-  depends_on "automake" => :build
   depends_on "hidapi"
-  depends_on "libftdi0"
-  depends_on "libhid"
-  depends_on "libusb-compat"
+  depends_on "libftdi"
+  depends_on "libusb"
```

Comment:
Build instructions mention using newer/maintained variants at https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-macOS and https://github.com/avrdudes/avrdude/wiki/General-Build-Instructions though may need to check if feature-complete.
- `libusb` vs `libusb-compat`
- `hidapi` vs `libhid`
- `libftdi` vs `libftdi0`

Reply:
BSD is GPL-compatible, so `elftoolchain` should be fine.